pkg/scheduler/schedulerpb/scheduler.proto

```diff
@@ -33,7 +34,11 @@ message SchedulerToQuerier {
   // Query ID as reported by frontend. When querier sends the response back to frontend (using frontendAddress),
   // it identifies the query by using this ID.
   uint64 queryID = 1;
-  httpgrpc.HTTPRequest httpRequest = 2;
+
+  oneof request {
+    httpgrpc.HTTPRequest httpRequest = 2;
```

Comment:
should this also be marked as deprecated?

Reply:
Done.